ppa/archive/gale.py

```diff
@@ -218,19 +226,23 @@ def get_item_pages(self, item_id, gale_record=None):
         if gale_record is None:
             gale_record = self.get_item(item_id)
 
+        # todo: needs try/catch error handling
+        local_ocr_text = get_local_ocr(item_id)
```

Comment:
This can throw an error if the underlying file doesn't exist right?

Reply:
yes, I was thinking to add similar error handling to what we implemented the first time